write-fonts/src/tables/cmap.rs

```diff
@@ -164,6 +164,35 @@ impl CmapSubtable {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum MappingError {
```

Comment:
if we only have a single error condition it's reasonable to just use a `struct` as the error type, which will simplify this somewhat.


edit: and to bikeshed, I would maybe just call that type `InvalidMapping`? or `CmapConflict`? 🤷

Reply:
Done